pkg/apisix/cluster.go

```diff
@@ -322,6 +325,7 @@ func (c *cluster) syncSchema(ctx context.Context, interval time.Duration) {
 	for {
 		if err := c.syncSchemaOnce(ctx); err != nil {
 			log.Warnf("failed to sync schema: %s", err)
+			c.metricsCollector.IncrSyncOperation("schema", "error")
```

Comment:
Maybe `failure`? success is corresponding to failure.

Reply:
done.